cypress/support/step_definition/help_section/commonHelpSectionRelatedSteps.spec.js

```diff
@@ -0,0 +1,19 @@
+import Help_Common from '../../pages/help_section/pageCommonHelpSection'
```

Comment:
I find the deviation strange. Here you only check sidebar stuff, but the links in the sidebar would also count as helpAreaSteps and vice versa.
For example does
Then('I can see the help articles page', () => {
  help.seeHelpArticlesPage()
})
also check what happens when 
When('I go to help articles in sidebar', () => {
  helpCommon.navigateToHelpArticles()
})
is done. 🤷‍♀️